client/src/features/viewers/ViewWrapper.jsx

```diff
@@ -222,8 +237,10 @@ const withSocket = (Component) => {
       ...timer,
       finished: playback === 'start' && timer.isNegative && timer.startedAt,
       clock: stringFromMillis(timer.clock),
+      clock12: format(new Date(timer.clock - hoursToMilliseconds(1)), 'hh:mm:ss a'),
       clockMs: timer.clock,
       clockNoSeconds: stringFromMillis(timer.clock, false),
+      clock12NoSeconds:  format(new Date(timer.clock - hoursToMilliseconds(1)), 'hh:mm'),
```

Comment:
I think we should not have this here. We should pass down only the `timer` and format it at the very point it is used in a given format.